packages/expo-gl-cpp/cpp/UEXGL.cpp

```diff
@@ -1,3057 +1,17 @@
 #include "UEXGL.h"
 
-#ifdef __ANDROID__
```

Comment:
If my memory is correct, this file was purposefully prefixed with `U` so it doesn't get versioned or sth like that? Can we name files `^E…` now? Will `expo-gl-cpp` version properly?

Reply:
UEXGL.h had to be prefixed because it's included from objective-c code, other files don't need to keep that naming
@tsapeta can you confirm?

Reply:
Yeah I can confirm, `expo-gl-cpp` is one of packages that are not being versioned so we need to prefix refs with `U` only for things that are being used in `expo-gl` itself and taking into account that only `UEXGL.h` methods are used in Objective-C/JNI we don't need to prefix other files.